po/residualvm/de_DE.po

```diff
@@ -7,86 +7,93 @@ msgid ""
-"POT-Creation-Date: 2017-11-05 14:10+0100\n"
-"PO-Revision-Date: 2017-11-05 14:44+0100\n"
-"Last-Translator: Lothar Serra Mari <rootfather@scummvm.org>\n"
+"POT-Creation-Date: 2019-04-27 21:32+0200\n"
+"PO-Revision-Date: 2019-04-27 22:16+0200\n"
+"Last-Translator: Lothar Serra Mari <serra@scummvm.org>\n"
 "Language-Team: Lothar Serra Mari <rootfather@scummvm.org>\n"
 "Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=ISO-8859-1\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: Poedit 2.0.4\n"
+"X-Generator: Poedit 2.2.1\n"
 
-#: gui/launcher.cpp:141 backends/platform/sdl/macosx/appmenu_osx.mm:106
+#: gui/launcher.cpp:142 backends/platform/sdl/macosx/appmenu_osx.mm:193
 msgid "Quit ResidualVM"
 msgstr "ResidualVM beenden"
 
-#: gui/launcher.cpp:142 backends/platform/sdl/macosx/appmenu_osx.mm:80
+#: gui/launcher.cpp:143 backends/platform/sdl/macosx/appmenu_osx.mm:187
 msgid "About ResidualVM"
 msgstr "�ber ResidualVM"
 
-#: gui/launcher.cpp:143
+#: gui/launcher.cpp:144
 msgid "Change global ResidualVM options"
 msgstr "Globale ResidualVM-Optionen �ndern"
 
-#: gui/launcher.cpp:529
+#: gui/launcher.cpp:508
 msgid ""
 "ResidualVM could not find any engine capable of running the selected game!"
 msgstr "ResidualVM kann keine f�r diese Spiel geeignete Engine finden!"
 
-#: gui/options.cpp:1091
+#: gui/options.cpp:1130
 msgid "Preserve aspect ratio"
 msgstr "Seitenverh�ltnis beibehalten"
 
-#: gui/options.cpp:1091
+#: gui/options.cpp:1130
 msgid "Preserve the aspect ratio in fullscreen mode"
 msgstr "Seitenverh�ltnis im Vollbildmodus beibehalten"
 
-#: gui/options.cpp:1094
+#: gui/options.cpp:1133
 msgid "V-Sync"
 msgstr "V-Sync"
 
-#: gui/options.cpp:1094
+#: gui/options.cpp:1133
 msgid "Wait for the vertical sync to refresh the screen"
 msgstr "Beim Bildaufbau auf vertikale Synchronisation warten"
 
-#: gui/options.cpp:1096
+#: gui/options.cpp:1135
 msgid "Game Renderer:"
 msgstr "Spiele-Renderer:"
 
-#: gui/options.cpp:1666
+#: gui/options.cpp:1729
 msgid "Language of ResidualVM GUI"
 msgstr "Sprache der ResidualVM-Oberfl�che"
 
-#: gui/options.cpp:1694
+#: gui/options.cpp:1756
+msgid ""
+"When starting a game, change the GUI language to the game language.That way, "
+"if a game uses the ResidualVM save and load dialogs, they are in the same "
+"language as the game."
+msgstr ""
+"Die Sprache der Men�f�hrung wird beim Starten des Spiels auf die Sprache des "
+"Spiels gestellt. Damit ist sichergestellt, dass bei der Verwendung der "
+"ResidualVM-eigenen \"Speichern und Laden\"-Funktionalit�t gleiche Sprachen "
+"verwendet werden."
+
+#: gui/options.cpp:1770
+msgid ""
+"Use the native system file browser instead of the ResidualVM one to select a "
+"file or directory."
+msgstr ""
+"Verwende zur Auswahl einer Datei oder eines Verzeichnisses den "
+"betriebssystemeigenen Dateimanager anstelle der ResidualVM-eigenen "
```

Comment:
Typo...des (anstatt der)
Except the "Auswahlmenüs" later on are meant to be majority, but then it wouldn´t fit with the earlier "den ... Dateimanager" as this is single aswell?
English seems to be single form in both cases.

German:
Sollte das nicht "des ... Auswahlmenüs" heißen?
Außer die "Auswahlmenüs" sind als Mehrzahl gedacht, dann aber passt das vorherige "den ... Dateimanager" nicht, weil der auch Einzahl ist?
Im Original sind beides Einzahl, soweit ich das verstehe.

Reply:
Stimmt, ist tatsächlich etwas inkonsistent, schließlich ist der Dateimanager das einzige "Betriebssystem-Auswahlmenü". Passe ich dann auch gleich in ScummVM an, da ist es genauso "falsch".